[2022-09-20 23:52:08,091] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: example_bash_operator.run_after_loop manual__2022-09-20T23:46:34.020872+00:00 [queued]>
[2022-09-20 23:52:08,118] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: example_bash_operator.run_after_loop manual__2022-09-20T23:46:34.020872+00:00 [queued]>
[2022-09-20 23:52:08,118] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2022-09-20 23:52:08,118] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2022-09-20 23:52:08,118] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2022-09-20 23:52:08,146] {taskinstance.py:1377} INFO - Executing <Task(BashOperator): run_after_loop> on 2022-09-20 23:46:34.020872+00:00
[2022-09-20 23:52:08,152] {standard_task_runner.py:52} INFO - Started process 7209 to run task
[2022-09-20 23:52:08,156] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'example_bash_operator', 'run_after_loop', 'manual__2022-09-20T23:46:34.020872+00:00', '--job-id', '16', '--raw', '--subdir', '/home/***/.local/lib/python3.7/site-packages/***/example_dags/example_bash_operator.py', '--cfg-path', '/tmp/tmpkskkztja', '--error-file', '/tmp/tmpwv_ypwc6']
[2022-09-20 23:52:08,157] {standard_task_runner.py:80} INFO - Job 16: Subtask run_after_loop
[2022-09-20 23:52:08,243] {task_command.py:369} INFO - Running <TaskInstance: example_bash_operator.run_after_loop manual__2022-09-20T23:46:34.020872+00:00 [running]> on host 3bbbf421189d
[2022-09-20 23:52:08,617] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=example_bash_operator
AIRFLOW_CTX_TASK_ID=run_after_loop
AIRFLOW_CTX_EXECUTION_DATE=2022-09-20T23:46:34.020872+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-09-20T23:46:34.020872+00:00
[2022-09-20 23:52:08,620] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-09-20 23:52:08,621] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo 1']
[2022-09-20 23:52:08,643] {subprocess.py:85} INFO - Output:
[2022-09-20 23:52:08,648] {subprocess.py:92} INFO - 1
[2022-09-20 23:52:08,649] {subprocess.py:96} INFO - Command exited with return code 0
[2022-09-20 23:52:08,948] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=example_bash_operator, task_id=run_after_loop, execution_date=20220920T234634, start_date=20220920T235208, end_date=20220920T235208
[2022-09-20 23:52:09,116] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-09-20 23:52:09,190] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
